src/sort.rs

```diff
@@ -56,6 +58,10 @@ fn by_date(a: &Meta, b: &Meta) -> Ordering {
     b.date.cmp(&a.date).then(a.name.cmp(&b.name))
 }
 
+fn by_version(a: &Meta, b: &Meta) -> Ordering {
+    compare(&a.name.name, &b.name.name).then(compare(&b.name.name, &a.name.name))
```

Comment:
Any specific reason why you have a `.then` chained onto it?

Reply:
my bad, this is a mistake. updated